.pre-commit-config.yaml

```diff
@@ -23,4 +23,4 @@ repos:
   # - repo: https://github.com/PyCQA/isort
```

Comment:
Nit. On line 22 above this we may want to change `exclude: '^rta|^kql'` to `exclude: '^kql'`